.github/workflows/test-action-build-init.yml

```
@@ -35,11 +35,16 @@ jobs:
         fail-on-error:
           - true
           - false
+        provide-links-safely:
+          - |
```

```suggestion
          - ''
          - |
```

Need to also add this to the `name:` field to differentiate the jobs, but I can't do it until I get back to my dev environment because GH still doesn't allow suggestions on unchanged lines (line 17). I'd probably recommend it being a bool, so something like:
```
, link-targets=${{ matrix.provide-links-safely == '' }}
```

---

Implemented in 8d376978132dbfe3c030303e2565332a23cba043 and 699d8f7ddc8c98b1d46199701429ac3fd032aa04. I've used the inverse condition.